cmd/server/main.go

```diff
@@ -545,7 +545,7 @@ func (p *Proxy) runHealthServer(o *ProxyRunOptions, server *server.ProxyServer)
 		fmt.Fprintf(w, "ok")
 	})
 	readinessHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, "ok")
```

Comment:
Isn't this going to return "true" rather than "ok"?

Reply:
Also for non "ok" responses we should be returning an error code via w.WriteHeader().
It needs to be called before we call write. We probably want to make it a 5XX error.